docs/program/sdks/_index.md

```diff
@@ -215,19 +253,27 @@ Find documentation on how to use these methods here:
 
 - [Arm](/components/arm/#api)
 - [Base](/components/base/#api)
+- [Board](/components/board/#api)
 - [Camera](/components/camera/#api)
+- [Encoder](/components/encoder/#api)
 - [Gantry](/components/gantry/#api)
 - [Gripper](/components/gripper/#api)
 - [Input Controller](/components/input-controller/#api)
 - [Motor](/components/motor/#api)
 - [Movement Sensor](/components/movement-sensor/#api)
 - [Sensor](/components/sensor/#api)
 - [Servo](/components/servo/#api)
-<!--
-Board API Docs should be added soon on Viam Documentation, + Encoder
--->
 
-You can find example code in the [Python SDK example GitHub repository](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/server/v1), the [Golang SDK example GitHub repository](https://github.com/viamrobotics/rdk/tree/main/examples), or the [TypeScript SDK example GitHub repository](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples).
+{{% alert title="Tip" color="tip" %}}
+
+You can find example code using the SDKs in the <file>examples</file> folder of the respective repository:
+
+- [Python SDK repository](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/server/v1)
+- [Go SDK repository](https://github.com/viamrobotics/rdk/tree/main/examples)
+- [TypeScript SDK repository](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples)
+- [Viam C++ SDK repository](https://github.com/viamrobotics/viam-cpp-sdk/tree/main/src/viam/examples)
```

Comment:
If you agree with above, recommend changing link titles to "X SDK examples" instead.